deb/after_install.sh

```
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+# Script which is run after install to ensure binary compatibility between
+# this and the previous version of libdrizzle-redux
+
+if [ -h /usr/lib/libdrizzle-redux.so.9 ]; then
+    echo "Removing outdated symlink"
+    sudo rm /usr/lib/libdrizzle-redux.so.10 /usr/lib/libdrizzle-redux.so.9 2>/dev/null
```

Do you need the `sudo` here? Won't the package be installed with superuser privileges anyway?

---

Presumably you used `2>/dev/null` to suppress errors if the sym links don't exist. If that was the only reason, I'd suggest using `rm -f` instead and avoid redirecting `stderr` to `/dev/null`.

---

> Do you need the sudo here? Won't the package be installed with superuser privileges anyway?
   
true sudo is not needed 

> resumably you used 2>/dev/null to suppress errors if the sym links don't exist

Indeed, I will change to your suggestion